core/src/test/scala/org/apache/spark/sql/delta/optimize/OptimizeMetricsSuite.scala

```diff
@@ -391,6 +410,7 @@ trait OptimizeMetricsSuiteBase extends QueryTest
         val dvStats = metrics.head.deletionVectorStats
         assert(dvStats.get.numDeletionVectorsRemoved === numFilesWithDVs)
         assert(dvStats.get.numDeletionVectorRowsRemoved === numDeletedRows)
+        assertOperationName(deltaLog)
 
         // Check DV metrics in the Delta history.
         val opMetrics = deltaTable.history.select("operationMetrics")
```

Comment:
select the name here and remove the `assertOperationName` which is making an additional call to deltaLog.history.